prawns/governance/src/handlers/technicalCommittee.Proposed.event.ts

```diff
@@ -0,0 +1,39 @@
+import { EventHandlerContext } from '@subsquid/substrate-processor';
+import { decodeAddress } from '../utils';
+import { SubstrateTechnicalCommitteeProposal, SubstrateNetwork } from '../model';
+import { getOrCreateGovernanceAccount } from '../utils';
+import { getTechnicalCommitteeProposedEvent } from './typeGetters/getTechnicalCommitteeProposedEvent';
+
+export default (network: SubstrateNetwork) =>
+  async (ctx: EventHandlerContext) => {
+    if (!ctx.event || !ctx.event.extrinsic) {
+      return;
+    }
+    const blockNumber = BigInt(ctx.block.height);
+    const date = new Date(ctx.block.timestamp);
+    const rootAccount = decodeAddress(ctx.event.extrinsic.signer);
+    const event = getTechnicalCommitteeProposedEvent(ctx, network);
+
+    const account = await getOrCreateGovernanceAccount(ctx.store, {
+      id: ctx.event.extrinsic.signer,
+      rootAccount,
+      network,
+    });
+    account.totalTechnicalCommitteeProposals = account.totalTechnicalCommitteeProposals + 1;
+    await ctx.store.save(account);
```

Comment:
I was just thinking about this. 
I guess there cannot be any race condition where we reach, in two separate events, at the `getOrCreateGovernanceAccount` at the same time and receive as `totalTechnicalCommitteeProposals` the same number. Then updating it with +1 will render a wrong total. 🤔

Reply:
Good question. I think the events have to be processed synchronously to guarantee order as otherwise you would not be able to calculate the current state. Might be worth checking that is the case though.

Reply:
They are sync. It's the major performance limitation of the processors, but they have to be else anything that we store representing mutable state would break and only logging actions would work